src/Common/Identifiers.hs

```diff
@@ -141,3 +150,13 @@ newtype FieldId = FieldId Identifier
 
 -- | A name to be bound; 'Nothing' represents a wildcard, e.g., @let _ = ...@
 type Binder = Maybe VarId
+
```

Comment:
This is how we classify whether an identifier identifies a constructor or a variable (for both types and data identifiers).